includes/multi-currency/MultiCurrency.php

```diff
@@ -845,6 +845,45 @@ public function get_price( $price, string $type ): float {
 		return $this->get_adjusted_price( $converted_price, $apply_charm_pricing, $currency );
 	}
 
+	/**
+	 * Gets a raw converted amount based on the amount and currency codes passed.
+	 * This is a helper method for external conversions, if needed.
+	 *
+	 * @param float  $amount        The amount to be converted.
+	 * @param string $to_currency   The 3 letter currency code to convert the amount to.
+	 * @param string $from_currency The 3 letter currency code to convert the amount from.
+	 *
+	 * @return float The converted amount.
+	 *
+	 * @throws InvalidCurrencyException
+	 */
+	public function get_raw_conversion( float $amount, string $to_currency, string $from_currency = '' ): float {
+		$enabled_currencies = $this->get_enabled_currencies();
+
+		// If the from_currency is not set, use the store currency.
+		if ( '' === $from_currency ) {
+			$from_currency = $this->get_default_currency()->get_code();
+		}
+
+		// We throw an exception if either of the currencies are not enabled.
+		$to_currency   = strtoupper( $to_currency );
```

Comment:
Thanks for doing this!

Reply:
No problem. You were right with what you said in your previous comment about how things were written, they were really simple and basic with no real forethought on errors. I added in the exceptions when the currencies are not about to be found because it makes sense since we can be dealing with anyone's input.